app/views/search/index.html.erb

```diff
@@ -1,11 +1,28 @@
 
 <div class="row row-content">
   <section>
-    <div id="collection_search_criteria">
-      <%= render partial: 'collection_search_criteria' %>
-    </div>
-    <div id="collection_search_results">
-      <%= render partial: 'collection_search_results' %>
+    <div id="search-criteria">
+      <%= render partial: 'search_criteria' %>
```

Comment:
I was already doing this with the two partials below, but since it was such a small partial I wasn't sure if the same principle should apply. Apparently yes, so I will make the change.